src/test/fixtures/node-worker.ts

```diff
@@ -21,6 +21,8 @@ function getWorkerScript(file: string): string {
       },
       postMessage: parentPort.postMessage.bind(parentPort),
       onmessage: function () {},
+      DecompressionStream,
+      CompressionStream,
```

Comment:
How did you figure out that this part was needed? I just hit the same issue for `Response` and I was not having a good time figuring it out.

Reply:
Haha, fair enough!